lib/util/BandUtil.js

```diff
@@ -48,18 +48,29 @@ export function updateParticipantBandKinds(activity, diBands) {
  */
 export function createParticipant(injector, attrs) {
   let participant = injector.get('bpmnFactory').create('bpmn:Participant', attrs);
-  participant.name = 'New Participant';
 
   let choreo = injector.get('canvas').getRootElement();
   if (choreo.businessObject.participants) {
     choreo.businessObject.participants.push(participant);
   } else {
     choreo.businessObject.participants = [ participant ];
   }
+  participant.name = 'New Participant ' + countNewParticipants(choreo.businessObject.participants);
   participant.$parent = choreo;
   return participant;
 }
 
+/**
+ *
+ * @param {Array} participants
+ */
+function countNewParticipants(participants) {
```

Comment:
Nice addition 😄 Can be tricked, though, if you for example manually rename `New Participant 2` to `New Participant 3` before adding the third. Not too relevant, though, users may trick themselves if they want to.

Reply:
Haha, yeah that is true. Also, having Names like "New Participant A" will lead to the numbers being _mixed _up__ but it is more of a convenience thing anyways.